src/cockpit/389-console/src/lib/plugins/passthroughAuthentication.jsx

```diff
@@ -71,6 +80,120 @@ class PassthroughAuthentication extends React.Component {
             urlEntryModalShow: false
         };
 
+        this.onExcludeToggle = isExcludedOpen => {
+            this.setState({
+                isExcludedOpen
```

Comment:
This is just nitpicking so ignore it if you like. Sometimes we use key: value and sometimes we use the ES6 way.

Reply:
This code above comes directly from patternfly 4 documentation.  We mix and match this styling up all over the code.  Do you think its something that needs to be fixed?

Reply:
> This code above comes directly from patternfly 4 documentation. We mix and match this styling up all over the code. Do you think its something that needs to be fixed?

Ye i noticed the different styling, no i think its fine as it is.

Reply:
LGTM